src/components/Hero/Hero.stories.mdx

```diff
@@ -0,0 +1,274 @@
+              />
+            }
+          />
+        </div>
+      )) ||
+      (args.heroType === HeroTypes.Tertiary && (
+        <Hero
+          {...args}
+          heading={
+            <Heading
+              level={1}
+              id={"1"}
+              text={"Hero Tertiary"}
+              blockName={"hero"}
+            />
+          }
+          subHeaderText="Explore our collection of hundreds of online resources and databases. Use our free online content to help with your research, whether it's finding a single article, tracing a family tree, learning a new language, or anything in between."
+        />
+      )) ||
+      (args.heroType === HeroTypes.FiftyFifty && (
+        <Hero
+          {...args}
+          subHeaderText="With 92 locations across the Bronx, Manhattan, and Staten Island, The New York Public Library is an essential part of neighborhoods across the city. Visit us today."
+          image={
+            <Image
+              src="https://placeimg.com/600/200/animals"
+              blockName={"hero"}
+              alt={""}
+            />
+          }
+          backgroundColor="#d0343a"
+          foregroundColor="#fff"
+        />
+      )) ||
+      (args.heroType === HeroTypes.Campaign && (
+        <Hero
+          {...args}
+          heading={
+            <Heading
+              level={1}
+              id={"1"}
+              text={"Hero Campaign"}
+              blockName={"hero"}
+            />
+          }
+          subHeaderText="With 92 locations across the Bronx, Manhattan, and Staten Island, The New York Public Library is an essential part of neighborhoods across the city. Visit us today."
+          backgroundImageSrc="https://placeimg.com/2400/800/nature/grayscale"
+          image={
+            <Image
+              src="https://placeimg.com/800/800/animals"
+              blockName={"hero"}
+              alt={""}
+            />
+          }
+        />
+      ))
+    }
+  </Story>
+</Preview>
+
+<ArgsTable story="Hero Props" />
+
+## Color Variations for Secondary Hero
+
+The background color for the title bar in HeroTypes.Secondary is determined by the placement of the `Hero` component. Below are the four color variations available based on the consuming React app.
+
+<Canvas>
+  <Story name="Color Variations for Secondary Hero">
+    <div className={"nypl-ds nypl--books-and-more"}>
+      <Hero
+        heroType={HeroTypes.Secondary}
+        heading={
+          <Heading
+            level={1}
+            id={"1"}
+            text={"Books and More"}
+            blockName={"hero"}
+          />
+        }
+        subHeaderText={
+          <div className="hero__body-text">
+            Explore our collection of hundreds of online resources and
+            databases. Use our free online content to help with your research,
+            whether it's finding a single article, tracing a family tree,
+            learning a new language, or anything in between.
+          </div>
```

Comment:
Since this gets repeated a few times (along with other props), make it a variable you can assign to all the different instances.

Reply:
Got it.  Thanks for the suggestion.  I cleaned up what I could.